bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -311,8 +318,12 @@ for tnum in trange:
                                                     (tnum - tmin, tmax - tmin))
 
 logging.info("Calculating median sigma for each template")
-median_sigma = [np.sqrt(np.median(trigf[args.ifo + '/sigmasq'][reg])) for reg in
-                    trigf[args.ifo + '/sigmasq_template'][:]]
+
+sigma_regions = trigf[args.ifo + '/sigmasq_template'][:]
+median_sigma = []
+for reg in sigma_regions:
```

Comment:
is this expected to be faster than the list comprehension or just easier to absorb as more shorter lines of code?

Reply:
I think it's just easier to read in this case.